### PR TITLE
Remove announcements and speed up deploy_nano script

### LINKED ISSUE #2795: Remove wifi/ethernet_announcements
### Description of the task
Since we're using static IP we should be able to remove wifi/ethernet announcements. Based on some initial testings, the announcements cause large spikes (i.e. slowdowns) in the runtime of thunderloop due to excessive SPI communication. They should be removed to make Thunderloop runtime more stable.

(Feel free to have a look at [this WIP branch](https://github.com/nimazareian/Software/tree/nima/debug_motion_planning) as it has some of the changes. Note that it also has many unrelated and untested changes).

### Acceptance criteria

- [ ] Remove wifi and ethernet announcements (Ansible playbooks will need to be updated) and update the necessary scripts: 
    - [ ] `run_ansible.py`
    - [ ] Add `display` to services in `deploy_nano.yml`
    - [ ] Other affected files...
- [ ] Confirm that Ansible uploads still work correctly without any errors. 
- [ ] Test the runtime of Thunderloop before and after changes and record the difference

### PR DESCRIPTION
**Pull request by LiCody**
### Description

- Deleted all references to python announcement or broadcast
  - Difference in thunder loop times https://docs.google.com/spreadsheets/d/1mZpd3lJ1awqCcQAwg7MaIsobw6PJFLBJK9FsYKIwoeo/edit#gid=0
  - services are stuck on robots so added stop and delete to deploy and setup nano respectively
- Deploy nano is mostly delayed by stopping robots
  - Lowered the timeout to 15s (default 90s)
- pybind with bazel doesnt work with cross compile

### Testing Done

- pinging works
- deploy_nano stopping services works 
- setup_nano deletes services
  - requires a daemon-reload which is run by other commands
  - service still appears after deletion but isnt doing anything on udevadm. This does disappear after a reboot

### Resolved Issues

resolves #2795, #2796 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

**Review by PhilD71 — DISMISSED** [review]

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml [hunk 1]
```diff
@@ -88,13 +86,15 @@
...
+            name: "{{ service_name }}"
+            enabled: no
             masked: no
-            daemon_reload: yes
-            state: started
+            state: stopped
+          with_items: ["wifi_announcement", "ethernet_announcement"]
+          loop_control:
+            loop_var: service_name
+          tags: always
```
Comment (nimazareian) [inline src/software/jetson_nano/ansible/playbooks/deploy_nano.yml:100]: Is this here to stay? Or is the idea that after sometime (when all robots have disabled announcements service) we will remove this part of the playbook? similarly for the setup_nano change

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml [hunk 2]
```diff
@@ -5,20 +5,18 @@
...
     binary_paths:
       thunderloop: ../../thunderloop_main
-      wifi_announcement: ../../broadcasts/robot_broadcast_sender_zip.zip
       display: ../../display/robot_ui_zip.zip
 
   tasks:
 
     - debug:
-        msg: "[Robot ID = {{ inventory_hostname }}]"
+          msg: "[Robot ID = {{ inventory_hostname }}]"
```
Comment (nimazareian) [inline src/software/jetson_nano/ansible/playbooks/deploy_nano.yml]: curious if this was intended. I know yaml can be pretty sensitive with indentations, so wanted to double check.

**Review by nimazareian — COMMENTED** [review]
Overall looks good to me. Just had a couple of questions. 
Have you also had the opportunity to test these changes on the robot and verify that it still works as intended?
Also, is there a plan to add a new separate script, similar to the one in #2796?

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml (hunk 2 above)
Comment (LiCody) [reply src/software/jetson_nano/ansible/playbooks/deploy_nano.yml]: fixed

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml (hunk 1 above)
Comment (LiCody) [reply src/software/jetson_nano/ansible/playbooks/deploy_nano.yml:100]: after sometime we will remove this

Comment (nimazareian) [reply src/software/jetson_nano/ansible/playbooks/deploy_nano.yml:100]: Sounds good. can you create a ticket for removing this in the future (probably under the robot software project)

**Review by nimazareian — APPROVED** [review]
lgtm!

**Review by PhilD71 — APPROVED** [review]

**Pull request merged**